index.js

```diff
@@ -746,6 +748,50 @@ export default async function makeHyperFetch ({
     return serveFile(drive, path, isRanged)
   }
 
+  async function diffVersions (request) {
+    const url = new URL(request.url)
+    const { hostname, pathname: rawPathname } = url
+    const pathname = decodeURI(ensureLeadingSlash(rawPathname))
+
+    const parts = pathname.split('/')
+    const versions = parts[3]
+    let [oldVersion, newVersion] = versions.split(/\.\./).map(Number)
+    const realPath = ensureLeadingSlash(parts.slice(4).join('/'))
+
+    const drive = await getDrive(`hyper://${hostname}/`, true)
+
+    const resHeaders = {
+      Link: `<${url.href}>; rel="canonical"`
+    }
+
+    if (oldVersion > drive.version || newVersion > drive.version) {
+      return {
+        status: 405,
```

Comment:
```suggestion
        status: 404,
```

Let's do "not found" instead of "not acceptable"